docs/technical/hooks.md

```diff
@@ -249,6 +249,22 @@ Implementing a hook should be made in consideration of the expected performance
 } );
 </pre>
 
+## 3.0
+
+### SMW::GetPreferences
+
+* Version: 2.5
```

Comment:
huh?

Reply:
shh ... this happens when you navigate different screens while responding to emails.